src/helpers/schema.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE subscribers (
   email VARCHAR(256) NOT NULL,
   address VARCHAR(256) NOT NULL,
   created BIGINT NOT NULL,
+  verified BIGINT,
```

Comment:
This can be `INT`, and should not be `NULL`
```suggestion
  verified INT NOT NULL,
```

Reply:
`verified` is similar to `created`, and recording the verification date.

It just give one more data, and fulfill the same purpose.

Should we downgrade to a simple 0|1 ?

Reply:
Ok we can keep timestamp like created but i would remove NULL
```suggestion
  verified BIGINT NOT NULL,
```